man/systemd.time.xml

```diff
@@ -108,37 +108,52 @@
   <refsect1>
     <title>Parsing Timestamps</title>
 
-    <para>When parsing, systemd will accept a similar syntax, but expects no timezone specification, unless
-    it is given as the literal string <literal>UTC</literal> (for the UTC timezone), or is specified to be
-    the locally configured timezone, the timezone name in the IANA timezone database format,
-    or in the <ulink url="https://tools.ietf.org/html/rfc3339">RFC 3339</ulink> profile of ISO 8601, as
-    <literal>Z</literal> or <literal>+<replaceable>05</replaceable>:<replaceable>30</replaceable></literal>
-    appended directly after the timestamp. The complete
-    list of timezones supported on your system can be obtained using the <literal>timedatectl
+    <para>When parsing, systemd will accept a similar syntax, but some fields can be omitted,
+    and the space between the date and time can be replaced with a <literal>T</literal>
+    (à la the <ulink url="https://tools.ietf.org/html/rfc3339">RFC 3339</ulink> profile of ISO 8601);
+    thus, in CET, the following are all identical:
+    <literal>Fri 2012-11-23 23:02:15 CET</literal>,
+    <literal>Fri 2012-11-23T23:02:15</literal>,
+    <literal>2012-11-23T23:02:15 CET</literal>,
+    <literal>2012-11-23 23:02:15</literal>.</para>
```

Comment:
I am not convinced thefact that this accepts to RFC 3399 should be mentioned in the first sentence.

I don't think RFC 3399 is a user friendly concept, the T/Z thing is not a syntax any regular human would ever want to use. 

RFC 3399 should be mentioned as alternative syntax, but not as first thing, but as an additional comment further down. Let's now drown people in cryptic, alternative syntaxes.

I think RFC 3399 might be OK for programs exchanging calendar times, but I don't think it's a user-friendly, and our docs should primarily focus on the user-friendly syntaxes, and then mention the more cryptic program-facing syntaxes further down.

Reply:
Germans calling me irregular counter: 2 today. Bad look.

Why you think replacing a space with a T renders a format so insanely unusable that no-one would ever write it, in face of years of people complaining that you don't support their favourite format, is beyond me.

Do you have some innovative way to group "we support `[%a ]%F{ |T}%T[ %Z]`" that doesn't make the manual even more confusing? I think I expressed it sanely. Of *course* it bears mentioning "hey we support both popular profiles of the standard timestamp format you read everywhere and write every day (unless you're lennart, in which case you only write the less popular profile)". Because if you want to say "we support `[%a ]%F %T[ %Z]`.            [other unrelated formats, timezone stuff]                                    oh also we support `[%a ]%FT%T[ %Z]`" then i don't really see how that's sensible

RFC3339 timezones are in the third paragraph. Your space-delimited format is in the second.

I don't think it takes up much space – may not be evident from reading the XML, so https://foreign.nabijaczleweli.xyz/pub/systemd.time.html#Parsing%20Timestamps – and it's mentioned as second-rate anyway ("you can replace it if you want").